tutor/specs/e2e/question-library.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { Mocker, visitPage, setTimeouts } from './helpers'
+
+describe('Question Library', () => {
+
+    Mocker.mock({
+        page,
+        routes: {},
+    })
+
+    beforeEach(async () => {
+        await setTimeouts()
+        await visitPage(page, '/course/1/questions')
+        await page.evaluate(() => {
+            window._MODELS.feature_flags.set('tours', false);
```

Comment:
is feature_flags needed?  mocker should do that?

Reply:
oh I didn't know Mocker takes care of that. Thanks.